src/cloudant/database.py

```diff
@@ -238,73 +235,29 @@ def get_view_result(self, ddoc_id, view_name, **kwargs):
-
-        :param str ddoc_id: Design document id used to get result.
-        :param str view_name: Name of the view used to get result.
-        :param bool descending: Return documents in descending key order.
-        :param endkey: Stop returning records at this specified key.  Can be
-            either a ``str`` or, for complex keys, a ``list``.
-        :param str endkey_docid: Stop returning records when the specified
-            document id is reached.
-        :param bool group: Using the reduce function, group the results to a
-            group or single row.
-        :param group_level: Only applicable if the view uses complex keys: keys
-            that are lists. Groups reduce results for the specified number
-            of list fields.
-        :param bool include_docs: Include the full content of the documents.
-        :param bool inclusive_end: Include rows with the specified endkey.
-        :param str key: Return only documents that match the specified key.
-        :param list keys: Return only documents that match the specified keys.
-        :param int page_size: Sets the page size for result iteration.
-        :param bool reduce: True to use the reduce function, false otherwise.
-        :param str stale: Allow the results from a stale view to be used. This
-            makes the request return immediately, even if the view has not been
-            completely built yet. If this parameter is not given, a response is
-            returned only after the view has been built.
-        :param startkey: Return records starting with the specified key.  Can be
-            either a ``str`` or, for complex keys, a ``list``.
-        :param str startkey_docid: Return records starting with the specified
-            document ID.
-
-        :returns: The result content wrapped in a Result object
-        """
-        view = View(DesignDocument(self, ddoc_id), view_name)
-        if kwargs:
-            return view.make_result(**kwargs)
-        else:
-            return view.result
-
-    def get_view_raw_result(self, ddoc_id, view_name, **kwargs):
-        """
-        Retrieves the raw JSON content based on the design document
-        and view name.  Unlike
-        :func:`~cloudant.database.CouchDatabase.get_view_result` the use
-        of ``skip`` and ``limit`` as kwargs is valid and
-        actually necessary in order to manage paging and iteration.
-        If you would like paging and iteration handled automatically for you
-        then use the
-        :func:`~cloudant.database.CouchDatabase.get_view_result`
-        method instead.
-
-        For example to retrieve the raw JSON response content based on a
+        To retrieve the raw JSON response content based on a
         design document view do:
 
         .. code-block:: python
 
-            db.get_view_raw_result('_design/ddoc_id_001', 'view_001')
+            db.get_view_result('_design/ddoc_id_001', 'view_001', True)
```

Comment:
Change this to `db.get_view_result('_design/ddoc_id_001', 'view_001', raw_result=True)`.  Yes you can set it directly as you have it but since this is an example I think it makes sense to explicitly use the `raw_result` argument name to underscore what is actually happening.